### DISCUSSION TITLE
ErrorBoundary for MetaFunction

### DISCUSSION THREAD

**Discussion by emilbryggare** [remix-run/remix#3297] [General]
I noticed that errors thrown in a MetaFunction for a route are not caught by ErrorBoundary. 

Should errors that are thrown by a MetaFunction be caught by the closest ErrorBoundary like for ActionFunction and LoaderFunction?

**Comment by sergiodxa** [comment]
There's an issue for this https://github.com/remix-run/remix/issues/3140

**Reply by emilbryggare** [reply]
Thanks, not sure how I missed that in my search. 🤷‍♂️
